pkg/utils/helpers.go

```diff
@@ -2,12 +2,14 @@ package utils
 
 import (
 	"bytes"
+	"context"
 	"encoding/json"
 	"errors"
 	"fmt"
 	"gopkg.in/yaml.v3"
 	"io"
 	"io/ioutil"
+	fp "path/filepath"
 	"spot-oceancd-cli/pkg/oceancd/model"
 	//"fmt"
```

Comment:
why do we have comments here?

Reply:
it isn't my change, I dunno :)
wanna remove it?